cypress/integration/editor/classic/custom-excerpt.spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Post custom excerpt', function () {
+	const postSetup = {
+		'title': 'Test Post',
+		'content': 'Some post content',
+		'excerpt': 'A custom post excerpt',
+		'url': null
+	};
+	it('Create new post named "' + postSetup.title + '".', function () {
+		cy.insertPost(postSetup.title, postSetup.content, 'post', false);
+		//Copies new post's url
+		cy.get('.post-publish-panel__postpublish-header a').
+		contains(postSetup.title).
+		should('have.attr', 'href').
+		then((href) => {
+			postSetup.url = href;
+		});
+	});
+
+	it('Sets up a custom Excerpt', function () {
+		cy.visit(postSetup.url);
+		cy.get('#wp-admin-bar-edit a').click();
+		cy.clearWelcome();
+		
+		//Selects the Excerpt panel
+        cy.get('div.components-panel__body').
+		contains('Excerpt').
+        click();
+		
+		//Sets the custom Excerpt and updates the post
+        cy.get('textarea.components-textarea-control__input').
+        click().
+        type(postSetup.excerpt);
+        cy.get( '.editor-post-publish-button' ).contains( 'Update' ).click();
+        cy.wait(2000);
```

Comment:
Is this wait necessary? It seems that you're trying to prep something for the next text, it would be better to set this up on the next text instead on a something like after hook inside the test. Check Cypress documentation regarding setting up state for test: https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks